README.md

```diff
@@ -5,6 +5,11 @@ This repository is currently under construction and will include the source code
 This paper proposes a network model composed of columns and layers that performs robust object learning and recognition. The model introduces a new feature to cortical columns, location information, which is represented relative to the object being sensed. Pairing sensory features with locations is a requirement for modeling objects and therefore must occur somewhere in the neocortex. We propose it occurs in every column in every region.
 > [Sources][1_src]
 
+### [The HTM Spatial Pooler – a neocortical algorithm for online sparse distributed coding][2]
```

Comment:
Capitalization:

The HTM Spatial Pooler—A Neocortical Algorithm for Online Sparse Distributed Coding

Reply:
Weird, their proof is!  Ok, leave it the way you have it.